storage/Validate/validateartifactupload.groovy

```diff
@@ -0,0 +1,29 @@
+import org.artifactory.build.promotion.PromotionConfig
+import org.artifactory.build.staging.ModuleVersion
+import org.artifactory.build.staging.VcsConfig
+import org.artifactory.exception.CancelException
+import org.artifactory.repo.RepoPathFactory
+import org.artifactory.request.Request
+import org.artifactory.util.StringInputStream
+
+storage {
+      beforeCreate { item ->
+        def ff = item.isFolder() ? 'folder' : 'file'        
```

Comment:
`ff` variable is not being used and should be removed.

Reply:
Removed